contracts/Liquidation.sol

```diff
@@ -321,13 +312,12 @@ contract Liquidation is ILiquidation, Ownable {
             amount,
             tracer.priceMultiplier(),
             liquidatedBalance.lastUpdatedGasPrice,
-            account,
-            market
+            account
         );
         
         // Limits the gas use when liquidating 
         require(
-            tx.gasprice <= uint256(IOracle(ITracerPerpetualSwaps(market).gasPriceOracle()).latestAnswer().abs()),
+            tx.gasprice <= uint256(IOracle(tracer.gasPriceOracle()).latestAnswer().abs()),
```

Comment:
This could be changed in a future PR, but I am pretty sure the tracer market does this check too. We should move the gasPriceOracle across to the Liquidations contract and remove it from Tracer

Reply:
Made ticket